composer.json

```diff
@@ -58,7 +58,7 @@
         "jms/serializer-bundle": "^0.13.0",
 
         "guzzlehttp/guzzle": "^5.2",
-        "ornicar/gravatar-bundle" : "1.1.2",
+        "ornicar/gravatar-bundle" : "1.1.3",
```

Comment:
We have only a few external dependencies, and not everyone follows properly the semantic version... so in that case, and because we can upgrade it by hand, I prefer this way.